charts/victoria-metrics-k8s-stack/Chart.lock

```diff
@@ -1,15 +1,15 @@
 dependencies:
 - name: victoria-metrics-operator
   repository: https://victoriametrics.github.io/helm-charts
-  version: 0.1.17
+  version: 0.1.19
```

Comment:
Looks good but could you please bump chart version

Reply:
what do you mean? 
`victoria-metrics-operator` chart used last available version. 

for this chart i bumped in [a3a3688](https://github.com/VictoriaMetrics/helm-charts/pull/185/commits/a3a3688e96126af65374ec2d7ee714f233386d7a)